### PR TITLE
created the class Corrector

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about to create the class Corrector :heavy_check_mark:

### CODE REVIEW

File: corrector.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    l = name.capitalize
+    l[0, 10]
```
Comment (dasileker) [inline corrector.rb]: - [ ] You don't have to create a new constant l, you already have name => name.capitalize and name[0, 10]

**Review by dasileker — CHANGES_REQUESTED** [review]

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @azy64,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: corrector.rb (hunk 1 above)
Comment (azy64) [reply corrector.rb]: That's the same! so why didn't you approve it ?

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**